#! /usr/bin/env python

PACKAGE='traversable_path'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()
#       Name       Type      Level Description     Default Min   Max
gen.add("diff_range_limit", double_t, 0, "Limit for the differential of the range.", 0.005, 0)
gen.add("diff_intensity_limit", double_t, 0, "Limit for the differential of the intensity.", 13.0, 0)
#TODO: this is unused in version 2 (or at least it is for the moment).
gen.add("min_path_width", double_t, 0, "Minimum width of a path to traverse it (unit: m)", 0.7, 0)

exit(gen.generate(PACKAGE, "traversable_path", "classify_terrain"))
